### PR TITLE
Fixes #9982: Document tag api on Rules and Directives

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/9982

### CODE REVIEW

File: src/api_v2/rule.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ https://github.com/Normation/rudder/blob/master/rudder-web/src/main/scala/com/no
    @apiParam (Mono) {Boolean} [enabled=true]     enabled the Rule at creation. But this parameter can be overriden : if you want to create Rule with directives and Rule targets directly with workflow enabled, the Rule will be created disabled (to prevent creating active Rule without validation). But this parameter can be overriden. Used to be 'isEnabled' in response, look <a href="#non-compliant-data-in-response">here</a> for more details.
    @apiParam (Multi) {UUID}       [directives] link one directive to the Rule. value needs to be an existing directive ID.
    @apiParam (Multi) {RuleTarget} [targets]    Link one target to the Rule. Rule targets are "group:groupId" and the special targets : "special:all" and "special:all_exceptPolicyServers".
+   @apiParam (Multi) {Tag}        [tags]       Tags linked to this Rule. they have a "key:value" format.
```
Comment (fanf) [inline src/api_v2/rule.md]: You need to add at least "see directive for example", or add examples

**Comment by amousset** [comment]
@VinceMacBuche This needs to be re-targeted to 4.1.

**Comment by VinceMacBuche** [comment]
PR rebased

File: src/api_v2/directives.md [hunk 2]
```diff
@@ -247,6 +247,7 @@ https://github.com/Normation/rudder/blob/master/rudder-web/src/main/scala/com/no
     @apiParam (Mono) {Boolean}           [enabled=true]                              Enable the Directive at creation. This parameter can be overriden. Used to be 'isEnabled' in response, look <a href="#non-compliant-data-in-response">here</a> for more details.
     @apiParam (Mono) {TechniqueVersion}  [techniqueVersion=latest Technique version] In which version your tehcnique should be created.
     @apiParam (Mono) {Int}               [priority]                                  Directive priority. Determines which value to use if there are several Directives based on the same technique applied to a Node.
+    @apiParam (Multi) {Tag}              [tags]                                      Tags linked to this directive. they have a "key:value" format.
```
Comment (fanf) [inline src/api_v2/directives.md]: I believe it is not exactly the case. I think that at some point, we changed it to an array of ... not sur. {"name":"...", "value":...}. Due to the the client side, I think?

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**